provider-gui/services/contract/config.ts

```diff
@@ -0,0 +1,49 @@
+import {
+    DatabaseTypes,
+    EnvironmentTypes,
+    NetworkNames,
+    NetworkPairTypeSchema,
+    ProsopoConfigSchema,
+} from '@prosopo/types'
+import { getLogLevel } from '@prosopo/common'
+
+const logLevel = getLogLevel()
+export const getConfig = (environment: EnvironmentTypes, network: NetworkNames) => {
+    return ProsopoConfigSchema.parse({
+        logLevel,
+        defaultEnvironment: environment,
+        defaultNetwork: network,
+        account: {
+            password: '',
+            address: 'currentAccount',
+        },
+        database: {
+            development: { type: DatabaseTypes.enum.mongoMemory, endpoint: '', dbname: 'prosopo_test', authSource: '' },
+        },
+        server: {
+            baseURL: 'http://localhost',
+            port: 9229,
+            fileServePaths: '[]',
+        },
+        networks: {
```

Comment:
This should default to sensible values if you don't supply it.

https://github.com/prosopo/captcha/blob/d5b42d16c2a10aec21a8d8b57a211a1f8e0dde07/packages/types/src/networks/index.ts#L10-L40

Reply:
Sorted